telephony/android_modem.c

```diff
@@ -1213,6 +1213,23 @@ amodem_send_stk_unsol_proactive_command( AModem  modem, const char* stkCmdPdu )
                           stkCmdPdu); //string type in hexadecimal character format
 }
 
+bool
+amodem_set_hold_feature( bool update, bool enable )
```

Comment:
To avoid declaring another global variable, I designed the function with a static variable, so I cannot separate the getter and the setter. However, can I extend the structure |AModemRec_| to record whether hold feature is supported?